Core/Src/debugleds.c

```diff
@@ -8,9 +8,12 @@
 
 #include "debugleds.h"
 #include "cmsis_os.h"
-#include "datalog_handler.h"
+#include "global_variables.h"
+#include "global_definitions.h"
 
 
+osMessageQueueId_t q_debugleds_messageHandle;
```

Comment:
essa variável tá definida na main.c, aqui ela é external
```suggestion
extern osMessageQueueId_t q_debugleds_messageHandle;
```